parm/post_avblflds.xml

```diff
@@ -8479,5 +8479,21 @@
        <scale>3.0</scale>
     </param>
 
+    <param>
+       <post_avblfldidx>1021</post_avblfldidx>
+       <shortname>VPOT_ON_ISOBARIC_SFC_FROM_WIND_FLD</shortname>
```

Comment:
@KarinaAsmar-NOAA To ensure consistency in the naming convention of  all pressure level variables, you might :
Change
  VPOT_ON_ISOBARIC_SFC_FROM_WIND_FLD
into
  VPOT_ON_ISOBARIC_SFC